Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 authors = ["Kisio Digital <team.coretools@kisio.com>", "Guillaume Pinot <texitoi@texitoi.eu>"]
 name = "transit_model"
-version = "0.32.2"
+version = "0.33.0"
```

Comment:
But you need to update all the _subcrates_ now :stuck_out_tongue:

Reply:
Oops :hand_over_mouth: 
Done 822464f9190dcccb79177d049c82f507bcf4455c